<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QDialog" name="SettingWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>287</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>267</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-image:url(&quot;:image/resource/image/skin_expand.png&quot;);
border:1px solid #e9e9e9;
border-radius:4px;
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>29</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>460</width>
     <height>271</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border: 2px solid gray;
     border-radius: 0px;
     padding: 0 8px;
     selection-background-color: darkgray;
}

QTabWidget::pane{border-top: 0px;
background-image: url(:image/resource/image/skin_basic.png);
}

QTabBar::tab{
color: 0;
font: bold 12px;
border: 0px solid #C4C4C3;
border-bottom-color: #C2C7CB;
min-width: 6ex;
min-height: 2ex;
padding: 3px;}
QTabBar::tab:selected{
color: rgb(204, 204, 204);
border: 0px solid #C4C4C3;
border-bottom-color: #C2C7CB;

color: 0;
font: bold 12px;
border-top-right-radius: 0px;
border-top-left-radius: 0px;
}
QTabBar::tab:!selected{
margin-top: 2px;
color: rgb(100, 100, 100);
background-image: url(:image/resource/image/skin_basic.png);}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_network">
    <attribute name="title">
     <string>  NetWorkSettings  </string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>12</y>
       <width>162</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>162</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>Preferenced vAccess Server:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>64</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Alternate vAccess Server:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>120</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Protocol:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_preferenceIp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>36</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_alternateIp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>87</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_port">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>80</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::NoEcho</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_protocol">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>120</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>http</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>https</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>263</x>
       <y>122</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_update">
    <attribute name="title">
     <string>  UpdateSettings  </string>
    </attribute>
    <widget class="QPushButton" name="pushBtn_check">
     <property name="geometry">
      <rect>
       <x>262</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(255, 255, 255);
font: bold 10pt;
border-radius: 0px;                                        
background-color: #669900;}                                       
QPushButton:hover{                                        
background-color: #77b200;}                                       
QPushButton:pressed{                                        
background-color: #99cc00;}  </string>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>140</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>You can check new version right now</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBtn_update_manual">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect manually  (by push the Check button below)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBtn_update_auto">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto detect(detect wether has version once runs vClient.
 Recommended)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBtn_update_auto2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto update (auto detect, update when detecting new
 version.(will restart system automatically))</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_ohter">
    <attribute name="title">
     <string>  Other Settings  </string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>97</y>
       <width>361</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>LounchWhenSystemStart</string>
     </property>
     <widget class="QCheckBox" name="checkBox_lounchAppOnSysStart">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>30</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Louch vClient when system start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>10</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="radioBtn_rdp_hasbar">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>0</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>show bar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>0</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RDP Bar:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtn_rdp_nobar">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>hide bar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>3</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FAP Bar:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtn_fap_hasbar">
      <property name="geometry">
       <rect>
        <x>106</x>
        <y>3</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>show bar</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBtn_fap_nobar">
      <property name="geometry">
       <rect>
        <x>211</x>
        <y>3</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>hide bar</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_tools">
    <attribute name="title">
     <string> NetworkCMD </string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_netparam">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>12</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushBtn_execute">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(255, 255, 255);
font: bold 10pt;
border-radius: 0px;                                        
background-color: #669900;}                                       
QPushButton:hover{                                        
background-color: #77b200;}                                       
QPushButton:pressed{                                        
background-color: #99cc00;}  </string>
     </property>
     <property name="text">
      <string>Excute</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_netcmdOutput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>43</y>
       <width>361</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCmd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>8</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>8</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/resource/image/corner_left_top.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color:#000000;/*#8bc600;*/
font: bold 10pt;</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>258</y>
     <width>285</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushBtn_ok">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
font: bold 12pt;
border-radius: 0px;                                        
background-color: #669900;}                                       
QPushButton:hover{                                        
background-color: #77b200;}                                       
QPushButton:pressed{                                        
background-color: #99cc00;}                                        </string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushBtn_apply">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
font: bold 12pt;
border-radius: 0px;                                        
background-color: #669900;}                                       
QPushButton:hover{                                        
background-color: #77b200;}                                       
QPushButton:pressed{                                        
background-color: #99cc00;}</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushBtn_cancel">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
font: bold 12pt;
border-radius: 0px;                                        
background-color: #669900;}                                       
QPushButton:hover{                                        
background-color: #77b200;}                                       
QPushButton:pressed{                                        
background-color: #99cc00;}  </string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>tabWidget</zorder>
  <zorder>layoutWidget_2</zorder>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_preferenceIp</tabstop>
  <tabstop>lineEdit_alternateIp</tabstop>
  <tabstop>comboBox_protocol</tabstop>
  <tabstop>lineEdit_port</tabstop>
  <tabstop>radioBtn_update_auto</tabstop>
  <tabstop>radioBtn_update_manual</tabstop>
  <tabstop>pushBtn_check</tabstop>
  <tabstop>radioBtn_rdp_hasbar</tabstop>
  <tabstop>radioBtn_rdp_nobar</tabstop>
  <tabstop>checkBox_lounchAppOnSysStart</tabstop>
  <tabstop>lineEdit_netparam</tabstop>
  <tabstop>pushBtn_execute</tabstop>
  <tabstop>textEdit_netcmdOutput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
